### DISCUSSION TITLE
Custom intro sequence

### DISCUSSION THREAD

**Discussion by Rekunan** [ppy/osu#15169] [Ideas] [answered]
Add an option to select a custom downloaded map/song that'll play after the "welcome to osu"
![](https://cdn.discordapp.com/attachments/826591073249525770/899429083228811274/unknown.png)

**Comment by peppy** [comment, marked as answer]
Not going to happen sorry.

**Reply by Rekunan** [reply]
saj, it's what it's

do you think you could say why?

**Reply by peppy** [reply]
Out of scope of the game.

**Reply by RocketSheepOriginal** [reply]
osu!supporter?
